app/models/concerns/with_qualifications.rb

```diff
@@ -39,7 +39,7 @@ module WithQualifications
     # subjects this course was tagged to.
     #
     # Defined here: https://github.com/DFE-Digital/manage-courses-api/blob/master/src/ManageCourses.Domain/Models/CourseQualification.cs
-    enum qualification: %i[qts pgce_with_qts pgde_with_qts pgce pgde]
+    enum qualification: { qts: 0, pgce_with_qts: 1, pgde_with_qts: 2, pgce: 3, pgde: 4 }
```

Comment:
Does this have any impact on the data at all? I'd rather skip linting on these rules to avoid any risk

Reply:
I think this is a good idea, no. This should make it harder to accidentally change the value for any of these enumerated value in the db.

Reply:
it does make a REAL diff, it infer int via defering it.


https://github.com/DFE-Digital/manage-courses-backend/pull/1026#discussion_r353704193

Reply:
According to [this](https://vaidehijoshi.github.io/blog/2015/02/03/when-enough-is-enough-how-to-know-when-to-use-enums/) these are equivalent, the only difference is explicit vs implicit. I should point out that this change was for the most part done automatically.